boot_services/src/boot_services.rs

```diff
@@ -1111,6 +1159,56 @@ impl BootServices for StandardBootServices<'_> {
         }
     }
 
+    fn set_watchdog_timer(&self, timeout: usize) -> Result<(), efi::Status> {
+        let set_watchdog_timer = self.efi_boot_services().set_watchdog_timer;
+        if set_watchdog_timer as usize == 0 {
+            panic!("function not initialize.")
+        }
+        match set_watchdog_timer(timeout, 0, 0, ptr::null_mut()) {
+            s if s.is_error() => Err(s),
+            _ => Ok(()),
+        }
+    }
+
+    fn stall(&self, microseconds: usize) -> Result<(), efi::Status> {
+        let stall = self.efi_boot_services().stall;
+        if stall as usize == 0 {
+            panic!("function not initialize.")
+        }
+        match stall(microseconds) {
+            s if s.is_error() => Err(s),
+            _ => Ok(()),
+        }
+    }
+
+    unsafe fn copy_mem_unchecked(&self, dest: *mut c_void, src: *const c_void, length: usize) {
+        let copy_mem = self.efi_boot_services().copy_mem;
+        if copy_mem as usize == 0 {
+            panic!("function not initialize.")
+        }
+        copy_mem(dest, src as *mut _, length);
+    }
+
+    fn set_mem(&self, buffer: &mut [u8], value: u8) {
+        let set_mem = self.efi_boot_services().set_mem;
+        if set_mem as usize == 0 {
+            panic!("function not initialize.")

```

Comment:
For clarity.

```suggestion
            panic!("boot services set_mem() function is not initialized.")
```

Reply:
Not going to leave a comment on each instance, but I think it'd be nice to apply that to each one.